Context/Object/ContentType.php

```
@@ -0,0 +1,244 @@
+    {
+        Assertion::assertTrue(
+            $this->checkContentTypeExistenceByIdentifier($identifier),
+            "Couldn't find a Content Type with identifier '$identifier'."
+        );
+    }
+
+    /**
+     * @Then Content Type (with identifier) :identifier does not exist
+     *
+     * Verifies that a content type with 'identifier' does not exist
+     */
+    public function assertContentTypeDoesntExistsByIdentifier($identifier)
+    {
+        Assertion::assertFalse(
+            $this->checkContentTypeExistenceByIdentifier($identifier),
+            "Found a Content Type with identifier '$identifier'."
+        );
+    }
+
+    /**
+     * @Then Content Type (with identifier) :identifier exists in Group with identifier :groupIdentifier
+     *
+     * Verifies that a content type with 'identifier' exists in group with identifier 'groupIdentifier'
+     */
+    public function assertContentTypeExistsByIdentifierOnGroup($identifier, $groupIdentifier)
+    {
+        Assertion::assertTrue(
+            $this->checkContentTypeExistenceByIdentifier($identifier, $groupIdentifier),
+            "Couldn't find Content Type with identifier '$identifier' on '$groupIdentifier."
+        );
+    }
+
+    /**
+     * Load and return a content type by its identifier
+     *
+     * @param  string  $identifier       content type identifier
+     * @param  boolean $throwIfNotFound  if true, throws an exception if it is not found.
+     *
+     * @return \eZ\Publish\API\Repository\Values\ContentType\ContentTypeGroup|null
+     */
+    protected function loadContentTypeByIdentifier($identifier, $throwIfNotFound = true)
+    {
+        try {
+            return $this->contentTypeService->loadContentTypeByIdentifier($identifier);
+        } catch (ApiExceptions\NotFoundException $e) {
+            return null;
+        }
+
+        if (!$contentType && $throwIfNotFound) {
+            throw new \Exception("Could not load content type with identifier '$identifier'");
```

You could just throw the exception again, it would avoid loosing the stacktrace.